.travis.yml

```diff
@@ -5,14 +5,13 @@ node_js:
   - 10.2.1
 env:
   matrix:
-    - MONGODB_VERSION=4.0.x MONGODB_TOPOLOGY=standalone
+    - MONGODB_VERSION=stable MONGODB_TOPOLOGY=standalone
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
     packages:
       - libkrb5-dev
-      - libstdc++6
```

Comment:
Why was this removed?

Reply:
Nice catch! Tests on travis were failing and I was experimenting with configuration. Forgot to return this back.